python/lsst/pipe/base/_check_qg_outputs.py

```diff
@@ -0,0 +1,216 @@
+# This file is part of pipe_base.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+from __future__ import annotations
+
```

Comment:
```suggestion
__all__ = (
    "QuantumGraphExecutionReport",
    "TaskExecutionReport",
    "DatasetTypeExecutionReport",
    "lookup_quantum_data_id",
)

```

All modules should have an `__all__` entries.  Among other things, that tells Sphinx which things should appear in the documentation.